packages/web3-utils/src/validation.ts

```diff
@@ -122,12 +122,12 @@ export const validateNumbersInput = (
 /**
  * checks input if typeof data is valid string input
  */
-export const isValidString = (data: any) => typeof data === 'string';
+export const isValidString = (data: Bytes) => typeof data === 'string';
```

Comment:
```suggestion
export const isValidString = (data: unknown) => typeof data === 'string';
```
For validation methods, we don't know the type of data being passed